app/views/handlers/question.py

```diff
@@ -82,6 +82,10 @@ def rendered_block(self) -> dict:
         )
 
         self._set_page_title(page_title)
+
+        if not transformed_block.get("question"):
+            return transformed_block
```

Comment:
Add a comment why we do this, as it might be confusing why question.py wouldn't have question. Ideally it could be to block.py